test/cqlpy/test_custom_index.py

```diff
@@ -0,0 +1,75 @@
+# Copyright 2025-present ScyllaDB
+#
+# SPDX-License-Identifier: LicenseRef-ScyllaDB-Source-Available-1.0
+
+###############################################################################
+# Tests for custom indexes
```

Comment:
Nitpick: it looks like this test file will be almost entirely on the the vector search (except just two test functions that I could find), and if we ever have a different type of custom index type, it will get its own tests, so perhaps you should just call this file test_vector_index.py ? 

I am guessing that in the future, you'll also have in this file the ability to skip most of the tests of that file if running on Scylla that is not configured with vector search enabled (remember that vector search needs Scylla to be configured **and** some external search engine needs to run, which not every developer running tests will want to do).